tar_scm.service

```diff
@@ -75,28 +75,28 @@
     </description>
   </param>
   <param name="filename">
-    <description>name of package - used together with version to determine tarball name</description>
+    <description>Name of package - used together with version to determine tarball name.</description>
```

Comment:
This description is a [noun phrase](http://en.wikipedia.org/wiki/Noun_phrase), which is grammatically inconsistent with the descriptions which begin "Specify ...", since the latter are [verb phrases](http://en.wikipedia.org/wiki/Verb_phrase).  If we really care about being consistent, we should choose one type and stick to it.